src/Symfony/Bridge/PhpUnit/Tests/DnsMockTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\DnsMock;
+
+require_once __DIR__.'/../DnsMock.php';
+
+class DnsMockTest extends \PHPUnit_Framework_TestCase
+{
+    protected function tearDown()
+    {
+        DnsMock::withMockedHosts(array());
+    }
+
+    public function testCheckdnsrr()
+    {
+        DnsMock::withMockedHosts(array('example.com' => array(array('type' => 'MX'))));
```

Comment:
In fact, people using the mock for checkdnsrr only have to define the type key. Adding the other keys would add boilerplate for them and may even confuse people ihmo.